src/os/bluestore/BlueStore.cc

```diff
@@ -5651,6 +5651,13 @@ int BlueStore::_open_bluefs(bool create, bool read_only)
   if (bluefs_layout.shared_bdev == BlueFS::BDEV_SLOW) {
 
     string options = cct->_conf->bluestore_rocksdb_options;
+    string options_annex = cct->_conf->bluestore_rocksdb_options_annex;
+    if (!options.empty() &&
+      !options_annex.empty() &&
+      *options.rbegin() != ',') {
+      options += ',';
+      options += options_annex;
+    }
```

Comment:
When `options_annex` is set, but `options` are empty, then `options_annex` are ignored. Is it an error?

Reply:
Ok, I guess you don't care when defaults are cleared.

Reply:
@aclamk - yep, you're right. Reworked it a bit.